docs/SPIR-V.rst

```diff
@@ -1541,7 +1542,8 @@ some system-value (SV) semantic strings will be translated into SPIR-V
 +---------------------------+-------------+----------------------------------------+-----------------------+-----------------------------+
 | SV_StencilRef             | PSOut       | ``FragStencilRefEXT``                  | N/A                   | ``StencilExportEXT``        |
 +---------------------------+-------------+----------------------------------------+-----------------------+-----------------------------+
-| SV_Barycentrics           | PSIn        | ``BaryCoord*AMD``                      | N/A                   | ``Shader``                  |
+| SV_Barycentrics           | PSIn        | ``BaryCoord*KHR`` for linear interp    | N/A                   | ``FragmentBarycentricKHR``  |
+|                           |             | ``BaryCoord*AMD`` for other types      |                       |                             |
```

Comment:
What are the "other types" here - did we add extra decorations for AMD interpolation modes, or are we translating core HLSL qualifiers to the AMD extension still? The KHR should support everything that HLSL does so we should be translating to that always unless we added extra bits for our extension.

Reply:
I will reword comment here.
The KHR extension adds two builtins for center_interp: BaryCoordKHR and BaryCoordNoPerspKHR.

From recent discussion, we need to support centroid_interp and sample_interp for MSAA too.
As KHR ext doesn't add extra builtins for this feature, I just want to keep existed AMD* builtins to meet the requirement.